draft-ppm-protocol.md

```diff
@@ -1142,6 +1145,10 @@ invalid input.
    shares.
 1. If computed over a sufficient number of input shares, output shares reveal
    nothing about either the inputs or the participating clients.
+1. Leaders can inject bogus "null" shares that do not affect the aggregate
+   output but mask the total number of shares to non-leader aggregators.
+1. Clients can inject bogus "null" shares that do not affect the aggregate
+   output but mask the total number of shares all aggregators.
```

Comment:
- The idea is pretty clear if you're as deep into this as we are, but I think someone new would have a hard time figuring out what a `bogus "null" share` is. We might stipulate that the core PPM protocol define this. If you'd like we can leave it as an open issue for #98.
- It sounds like what you want is that the clients or servers contribute "null" shares + some noise for DP. If this is correct, then let's say it.
- Instead of `inject bogus "null" shares` let's say "generate bogus reports that encode "null" shares`, as this is a bit more precise.

Reply:
> It sounds like what you want is that the clients or servers contribute "null" shares + some noise for DP. If this is correct, then let's say it.

No, the null shares should be effectively "no-ops", and not introduce any change in the visible output. They only serve to protect the helpers from learning the true count.

Your other suggestions sound good, I can make an update